include/multiverso/table_interface.h

```diff
@@ -30,25 +31,26 @@ class WorkerTable {
   int GetAsync(Blob keys, const GetOption* option = nullptr);
   int AddAsync(Blob keys, Blob values, const AddOption* option = nullptr);
 
-  void Wait(int id);
+  void Wait(integer_t id);
```

Comment:
id can be re-used, like the TCP sequence number. Moreover, we may don't want it consistent with the row index.